src/components/OnboardingComponents/SemanticSearch/QuerySuggestions.tsx

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {Box, Chip, Typography} from '@mui/material';
+import {useTranslation} from 'react-i18next';
+
+interface QuerySuggestionsProps {
+  /** A list of suggestion strings to display. */
+  suggestions: string[];
+  /** The function to call when a suggestion chip is clicked. */
+  onSuggestionClick: (query: string) => void;
+}
+
+/**
+ * A component that displays a list of clickable query suggestion chips.
+ */
+export default function QuerySuggestions({suggestions, onSuggestionClick}: QuerySuggestionsProps): JSX.Element {
+  const {t} = useTranslation('global');
+
+  return (
+    <Box mb={2}>
+      <Typography variant='caption' sx={{color: 'GrayText', mb: 1, display: 'block'}}>
```

Comment:
spread this out for better readability.
```suggestion
      <Typography
        variant='caption'
        sx={{
          color: 'GrayText',
          marginBottom: 1,
          display: 'block',
        }}
      >
```

Reply:
Done